### PR TITLE
Additions to Module 2 Hands-On

### PR DESCRIPTION
**Pull request by jack89roberts**
Haven't created solutions yet.

### CODE REVIEW

File: coursebook/modules/m2/2-hands-on.ipynb [hunk 1]
```diff
@@ -8,12 +8,14 @@
     "\n",
```
Comment (lannelin) [inline coursebook/modules/m2/2-hands-on.ipynb:8]: <p>Are we expecting this to be done by parsing the category file with code, like in the prewritten code "code for later (skip for now!)"? Or will we prompt this to be more manual?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/rds-course/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (lannelin) [inline coursebook/modules/m2/2-hands-on.ipynb:8]: <p>great idea</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/rds-course/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (lannelin) [inline coursebook/modules/m2/2-hands-on.ipynb:8]: <p>Great point with the values representing NA</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/rds-course/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

**Comment by lannelin** [comment]
Great additions. One outstanding q in the notebook but more for my info than any change requested.

File: coursebook/modules/m2/2-hands-on.ipynb (hunk 1 above)
Comment (jack89roberts) [reply coursebook/modules/m2/2-hands-on.ipynb:8]: I was thinking probably just more manual for this one, `.replace(values_with_dict_I_made)` or something, but they could try parsing the category file if they want to.

**Pull request merged**